_sass/blocks/explore/_subpage.scss

```diff
@@ -146,33 +145,34 @@
     }
   }
 
-  .subregion-chart {
-    display: inline-block;
-    float: right;
-    min-width: 200px;
-    position: relative;
-    top: 0.4em;
-    width: 33%;
+  table.subregions {
+    border-collapse: collapse;
+    width: 100%;
 
-    .bars {
-      display: block;
-      vertical-align: baseline;
+    &,
+    td,
+    th,
+    tr {
+      border: 0;
+      text-align: left;
     }
 
-    .label {
-      font-weight: $weight-light;
+    thead {
+      background: none;
+      border: 0;
     }
 
-    .negative .label {
-      font-size: 0.8em;
+    thead eiti-bar {
```

Comment:
Merge rule `thead eiti-bar` with rule on line 160<br>Selector should have depth of applicability no greater than 3, but was 4